api/components/com_contact/src/Serializer/ContactSerializer.php

```diff
@@ -70,7 +70,7 @@ public function category($model)
         $serializer = new JoomlaSerializer('categories');
 
         $resource = (new Resource($model->catid, $serializer))
-            ->addLink('self', Route::link('site', Uri::root() . 'api/index.php/v1/content/categories/' . $model->catid));
+            ->addLink('self', Route::link('site', Uri::root() . 'api/index.php/v1/contact/categories/' . $model->catid));
```

Comment:
```suggestion
            ->addLink('self', Route::link('site', Uri::root() . 'api/index.php/v1/contacts/categories/' . $model->catid));
```
there is no resource at `api/index.php/v1/contact/categories`

Reply:
in this page, Contact section
https://docs.joomla.org/J4.x:Joomla_Core_APIs#Get_List_of_Component_Configs#Contact
wrote about contact categories
https://docs.joomla.org/J4.x:Joomla_Core_APIs#Get_List_of_Component_Configs#Categories_2

Reply:
https://github.com/joomla/joomla-cms/blob/5.2-dev/plugins/webservices/contact/src/Extension/Contact.php#L74

If it doesn’t exist this code here is buggy